tests/functional/helper.js

```diff
@@ -568,17 +564,26 @@ module.exports = {
     _lastAddonLoadingCompleted = false;
 
     await this.setSetting(
-        'addonCustomServerAddress', `${addonServer.url}/${addonPath}/`);
+      'addonCustomServerAddress', `${addonServer.url}/${addonPath}/`);
```

Comment:
why all of these changes? I think the file is already indented via clang-format

Reply:
This is a JavaScript file, so clang won't do anything... I have a "Format on Save" setup on VSCode and I didn't notice if had reformatted so much.

Reply:
This is not right. clang-format works with JS perfectly. See: `./scripts/apply-format -f -i tests/functional/helper.js`

Reply:
Hm, TIL. I'll revert these changes, but we should be running these lints on CI.